### PR TITLE
Erased navigation and everything related to it

### PR DESCRIPTION
**Pull request by Allexander203**

### CODE REVIEW

File: tcms/urls.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
     re_path(r"^xml-rpc/", RPCEntryPoint.as_view(protocol=XMLRPC_PROTOCOL)),
     re_path(r"^json-rpc/$", RPCEntryPoint.as_view(protocol=JSONRPC_PROTOCOL)),
     re_path(
-        r"^navigation/", core_views.NavigationView.as_view(), name="iframe-navigation"
+
     ),
```
Comment (asankov) [inline tcms/urls.py:30]: you also need to delete the call to `re_path`

**Review by asankov — CHANGES_REQUESTED** [review]

File: tcms/urls.py [hunk 2]
```diff
@@ -25,9 +25,7 @@
     re_path(r"^$", core_views.DashboardView.as_view(), name="core-views-index"),
     re_path(r"^xml-rpc/", RPCEntryPoint.as_view(protocol=XMLRPC_PROTOCOL)),
     re_path(r"^json-rpc/$", RPCEntryPoint.as_view(protocol=JSONRPC_PROTOCOL)),
-    re_path(
-        r"^navigation/", core_views.NavigationView.as_view(), name="iframe-navigation"
-    ),
+
```
Comment (asankov) [inline tcms/urls.py:28]: delete the empty line

File: tcms/core/tests/test_views.py [hunk 3]
```diff
@@ -16,35 +16,7 @@
...
-    def _common_navigation_assertions(self, response):
-        self.assertContains(response, _("DASHBOARD"))
-        self.assertContains(response, _("TESTING"))
-        self.assertContains(response, _("SEARCH"))
-        self.assertContains(response, _("TELEMETRY"))
-        self.assertContains(response, _("ADMIN"))
+
 
 
 class TestDashboard(BaseCaseRun):
```
Comment (asankov) [inline tcms/core/tests/test_views.py:22]: you should have only 2 empty rows between class definitions

**Review by asankov — CHANGES_REQUESTED** [review]

File: tcms/core/tests/test_views.py [hunk 4]
```diff
@@ -8,43 +8,6 @@
 
 from tcms import urls
 from tcms.tests import BaseCaseRun
-from tcms.tests.factories import (
-    TestExecutionFactory,
-    TestPlanFactory,
```
Comment (atodorov) [inline tcms/core/tests/test_views.py:13]: Some of these imports are still in use, see the failing checks at https://github.com/kiwitcms/Kiwi/pull/2239/checks?check_run_id=1839431769 and https://github.com/kiwitcms/Kiwi/pull/2239/checks?check_run_id=1839432801 for example.

**Review by atodorov — CHANGES_REQUESTED** [review]

**Comment by atodorov** [comment]
Cleaned up in https://github.com/kiwitcms/Kiwi/pull/2247

**Pull request closed without merging**